src/components/Comments/ContainerComment.js

```diff
@@ -78,6 +78,12 @@ const ContainerComment = ({
       updateVote();
     }
   };
+  const handlePaddingBottom = () => {
+    if (!itemParent?.message || itemParent?.message === '') {
+      return calculatePaddingBtm() + calculationText(itemParent?.message).containerHeight;
+    }
+    return calculatePaddingBtm();
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic in the `handlePaddingBottom` function seems to be incorrect. If `itemParent?.message` is null or empty, it adds `calculationText(itemParent?.message).containerHeight` to `calculatePaddingBtm()`. However, if `itemParent?.message` is null or empty, `calculationText(itemParent?.message).containerHeight` will likely also be 0 or undefined, which may not be the intended behavior. Please review this logic.
```diff
-    if (!itemParent?.message || itemParent?.message === '') {
-      return calculatePaddingBtm() + calculationText(itemParent?.message).containerHeight;
-    }
+    if (itemParent?.message && itemParent?.message !== '') {
+      return calculatePaddingBtm() + calculationText(itemParent?.message).containerHeight;
+    }
     return calculatePaddingBtm();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->